server/routes/private.js

```diff
@@ -49,6 +50,41 @@ router.post('/api/book-appointment', async (req, res) => {
 
 });
 
+router.post('/api/recurring_schedule', async (req, res) => {
```

Comment:
I would suggest changing the endpoint to add-recurring-schedule as per suggested coding naming conventions. Since this endpoint would change the database, the verb would signify that the endpoint manipulates the database.

Reply:
Thanks @catai9 I've modified the end point to add_recurring_schedule